python/lib/eeg.py

```diff
@@ -671,9 +671,35 @@ def fetch_and_insert_event_file(
                 # get the blake2b hash of the task events file
                 blake2 = blake2b(event_file.path.encode('utf-8')).hexdigest()
                 # insert event data in the database
-                physiological.insert_event_file(
-                    event_data, event_path, physiological_file_id, blake2
-                )
+                # TODO: Temporary
+                # TODO: should not have two different insert_event_file methods
+                # TODO: remove the TRY/CATCH and the legacy method when
```

Comment:
I feel it is missing something after when in the comment ;)

Reply:
Yes, it is! Good catch :)